src/view/sidebar/edit-modal/parts/share-calendar-url-modal.tsx

```diff
@@ -14,15 +14,9 @@ import { UrlColumn } from './url-column';
 const getUrl = (type: string, user: string, folderName: string): string => {
 	const domain =
```

Comment:
can you also modify this line of code? It makes no sense to hardcode crb2
```suggestion
	const domain = window.location.hostname
```